modules/post/linux/gather/checkvm.rb

```diff
@@ -199,6 +205,37 @@ def run
         end
       end
     end
+    if !vm
+      xen_type = read_file('/sys/hypervisor/type')
+      if xen_type
+        if xen_type == 'xen'
+          vm = 'Xen'
```

Comment:
I found this for context here from [kernel.org](https://www.kernel.org/doc/Documentation/ABI/stable/sysfs-hypervisor-xen) :+1: 

> What:		/sys/hypervisor/type
Date:		March 2009
KernelVersion:	2.6.30
Contact:	xen-devel@lists.xenproject.org
Description:	If running under Xen:
		Type of hypervisor:
		"xen": Xen hypervisor